spec/models/task_spec.rb

```diff
@@ -1,7 +1,17 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-
-RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe 'タスク管理機能', type: :model do
+  it 'titleが空ならバリデーションが通らない' do
+    task = Task.new(task_name: '', content: '失敗テスト')
```

Comment:
FactoryBotの機能を使うと下記の様に記載できることも知っておいてください。ここでの記法はどちらでも大丈夫です。
```suggestion
    task = FactoryBot.build(:task, task_name: '', content: '失敗テスト')
```